letsencrypt/reporter.py

```diff
@@ -76,9 +77,10 @@ def print_messages(self):
         if not self.messages.empty():
             no_exception = sys.exc_info()[0] is None
             bold_on = sys.stdout.isatty()
-            if bold_on:
-                print(le_util.ANSI_SGR_BOLD)
-            print('IMPORTANT NOTES:')
+            if not self.config.quiet:
```

Comment:
This logic isn't quite right.

Before, `bold_on` was set and `le_util.ANSI_SGR_BOLD` was written to the terminal depending on the value of `bold_on`. Now, `bold_on` can be set to `True` but we never actually write `le_util.ANSI_SGR_BOLD` to the terminal, causing us to unnecessarily write `le_util.ANSI_SGR_RESET` later.

This isn't a big deal, but it should probably be written correctly.